pkg/modules/authorization_test.go

```diff
@@ -449,6 +449,27 @@ func TestAuthorizationServerPreCheck(t *testing.T) {
 
 			return true, auth, tmpCR, sourceClient, fakeControllerRuntimeClient
 		},
+		"success v2": func(*testing.T) (bool, csmv1.Module, csmv1.ContainerStorageModule, ctrlClient.Client, fakeControllerRuntimeClientWrapper) {
+			customResource, err := getCustomResource("./testdata/cr_auth_proxy.yaml")
+			if err != nil {
+				panic(err)
+			}
+
+			tmpCR := customResource
+			auth := tmpCR.Spec.Modules[0]
+
+			karaviConfig := getSecret(customResource.Namespace, "karavi-config-secret")
+			karaviTLS := getSecret(customResource.Namespace, "karavi-auth-tls")
```

Comment:
if this is the secret used for certificates, the name changed to karavi-selfsigned-tls

Reply:
Updated.